examples/analytics-error-reporting.amp.html

```diff
@@ -72,12 +72,12 @@ <h2 id="amp-iframe">IFrame</h2>
     <script type="application/json">
         {
             "requests": {
-                "user-error": "http://localhost:8000/dsdsdsd"
+                "error": "http://localhost:8000/dsdsdsd"
```

Comment:
Is it possible to have this request include detail on what the error is? If so, then it should be included as a substitution variable and documented.

Reply:
I think so yeah, let me try that and let you know!

Reply:
Can the request be updated to something like

`http://localhost:8000/ping?en=${errorName}&em=${errorMessage}`